### DISCUSSION TITLE
Add option to have filters visible by default (slide-down only)

### DISCUSSION THREAD

**Discussion by kungro** [rappasoft/laravel-livewire-tables#997] [Feature Requests]
It would be nice to have an option to have the filters visible when loading the table. Something like setFiltersVisibilityLoadStatus(bool).

Thanks.

**Comment by lrljoe** [comment]
Nice thought!  In the meantime you can publish the views and edit 
components\wrapper.blade.php 

Make the following modification

`x-data="{ filtersOpen: true }"`

`x-data="{ filtersOpen: $wire.entangle('slideOpenByDefault') }"`

Then set yourself up a public variable in your table instance:
`public bool $slideOpenByDefault = true;`

I've opened PR #998 which should help with this

**Comment by lrljoe** [comment]
This should be in the next release, [see here for more info](https://github.com/rappasoft/laravel-livewire-tables/pull/1017)

**Comment by rappasoft** [comment]
Available in v2.10
